modules/olm-operatorgroups-troubleshooting.adoc

```diff
@@ -9,5 +9,21 @@
 [id="olm-operatorgroups-troubleshooting-membership_{context}"]
 === Membership
 
-* If more than one Operator group exists in a single namespace, any CSV created in that namespace transitions to a failure state with the reason `TooManyOperatorGroups`. CSVs in a failed state for this reason transition to pending after the number of Operator groups in their namespaces reaches one.
+* When attempting to generate a cluster service version (CSV) in a namespace, an install plan considers an Operator group invalid in the following scenarios:
++
+--
+** No Operator groups exist in the install plan's namespace.
+** Multiple Operator groups exist in the install plan's namespace.
+** An incorrect or non-existent service account name is specified in the Operator group.
+--
```

Comment:
Can you add a note or sentence that the install plan's namespace must contain only one Operator group? Unless that is stated elsewhere. I was momentarily confused by the errors on _No_ and _Multiple_.